### DISCUSSION TITLE
crop=True vs crop=scale

### DISCUSSION THREAD

**Discussion by mogoh** [SmileyChris/easy-thumbnails#570] [Q&A]
Hello,

So this works:
```
<img
src="{% thumbnail news.picture "1240x766" crop=True upscale %}"
/>
```

And this does not:
```
<img
src="{% thumbnail news.picture "1240x766" crop=scale upscale %}"
/>
```

Why does `crop=scale` not work?
Where is `crop=True` documented?
Is this a bug or a feature?

**Comment by jrief** [comment]
If you use `{% thumbnail news.picture "1240x766" crop=scale %}` then `scale` has to be defined inside your rendering context. It can be either `True` or `False`.

Does this answer your question?

**Reply by mogoh** [reply]
Well, no.
The [documentation](https://easy-thumbnails.readthedocs.io/en/latest/usage/#thumbnail-options) suggests:

> `crop=<smart|scale|W,H>` cuts the edges of the image to match the aspect ratio of size before resizing.

And [later](https://easy-thumbnails.readthedocs.io/en/latest/usage/#thumbnail-tag):

> options are a space separated list of options which are used when processing the image to a thumbnail such as `sharpen`, `crop` and `quality=90`.

How I read this:

Options could be `crop=scale` or `crop`, not `crop=True`.

I get it now: `crop=True` is the right way, but the documentation is misleading.
Or am I overlooking something?
